aws/templates/docker_compose_xchain.tftpl

```diff
@@ -17,6 +17,8 @@ services:
             - index
             - --config-file=/config/${xchain_config}.json
             - --omni-rpc=${omni_rpc}
+            volumes:
+            - /config/config.json:/config/config.json
```

Comment:
mapping to config.json on purpose, to preserve backward compatibilty by not overriding preexisting config files (staging.json, testnet.json). These files will be ditched once we test on staging.

Reply:
nit: config files are generally stored in `/etc`, so I suggest `/etc/xchain/config.json`

Reply:
if you agree, i would introduce this change after i test that the current setup works on staging. Would like to minimize changes as much as possible.